app/src/main/res/layout/activity_main.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The report told me that the frame layout was unnecessary at that location. I also thought it was a bit weird to do at the activity_main, for content_main it is understandable.